### PR TITLE
binutils-2.26: Inhibit -Werror as new system GCC has case fallthrough warnings

### PR DESCRIPTION
**Pull request by zao**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/b/binutils/binutils-2.26.eb [hunk 1]
```diff
@@ -14,6 +14,8 @@ checksums = [
     '1fd3088df8089d269c1e6ab74c5b4eb7793f9a321515ef96420f0d5ffef3fe75',  # binutils-2.26_GCC-6.patch
 ]
 
+configopts = ' --disable-werror'
```
Comment (boegel) [inline easybuild/easyconfigs/b/binutils/binutils-2.26.eb]: @zao Please add a comment to explain why this is there ;)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2070.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/2832703429e0141d52ed4cf5f4cc0bb0 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2482.golett.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/7dd982443a9e2eab2a0a865c106ef9e4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3107.skitty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/c15bd4602566140f806d1c6379814fac for a full test report.

**Comment by boegel** [comment]
Hmm, seems like the Travis notification to GitHub didn't pass through correctly, closing & re-opening to try and fix that...

**Comment by murfreesboro** [comment]
Test report by @murfreesboro
**SUCCESS**
Build succeeded for 0 out of 0 (1 easyconfigs in this PR)
phoenix - Linux ubuntu 16.04, Intel(R) Core(TM) i5-7300HQ CPU @ 2.50GHz, Python 2.7.12
See https://gist.github.com/3c02d509bf97048afccf1811356c3879 for a full test report.

**Comment by boegel** [comment]
@murfreesboro Please use `--force` (and a temporary install location) when submitting test reports

**Comment by vanzod** [comment]
Going in, thanks @zao!

**Pull request merged**